src/components/form/Textarea/textarea.module.css

```diff
@@ -16,13 +16,14 @@
 }
 
 .textarea:focus:enabled {
-    border: 2px solid var(--amp-colors-focus-border);
+    border: 2px solid var(--amp-colors-input-border-focus);
     outline: none;
 }
 
 .textarea:disabled {
-    background-color: var(--amp-colors-bg-highlight);
-    border: 1px solid var(--amp-colors-neutral-400);
```

Comment:
we need to remove hard coded values to the color palette and use semantic variables to support dark mode.